### PR TITLE
Add Vector to the Airflow images

### PR DESCRIPTION
**Pull request by siegfriedweber**
Add Vector to the Airflow images

Prerequisite for stackabletech/airflow-operator#213

### CODE REVIEW

**Review by maltesander — APPROVED** [review]
LGTM.

File: airflow/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6@sha256:c5ffdf5938d73283cec018f2adf59f0ed9f8c376d93e415a27b16c3c6aad6f45 AS airflow-build-image
```
Comment (nightkr) [inline airflow/Dockerfile:1]: Should we move vector out of the sandbox now that we have "top-level" images depending on it?

Comment (siegfriedweber) [reply airflow/Dockerfile:1]: Yes, this was on my TODO list before merging. Usually I add this TODO list to the description of the pull request but I forgot it this time.

File: airflow/Dockerfile [hunk 2]
```diff
@@ -60,32 +60,24 @@ RUN microdnf install -y yum python${PYTHON} \
 ENV HOME=/stackable
 ENV AIRFLOW_USER_HOME_DIR=/stackable
 ENV PATH=$PATH:/bin:$HOME/app/bin
-ENV AIRFLOW_UID="50000"
+ENV AIRFLOW_UID="1000"
```
Comment (nightkr) [inline airflow/Dockerfile:63]: Does this require a manual migration step for users?

**Review by maltesander — APPROVED** [review]
LGTM.

File: airflow/Dockerfile (hunk 2 above)
Comment (siegfriedweber) [reply airflow/Dockerfile:63]: The data should not be affected by this change. But the UID stored in the airflow-operator must be in sync with this one. So the airflow-operator 23.4.0 is only compatible with the stackable23.4.0 Airflow image.

Comment (nightkr) [reply airflow/Dockerfile:63]: Alright, fair enough.

**Review by nightkr — APPROVED** [review]

**Pull request merged**